automation_tools/__init__.py

```diff
@@ -1015,12 +1015,9 @@ def setup_foreman_discovery(sat_version):
         return
 
     if sat_version == 'upstream-nightly':
-        run('foreman-installer --enable-foreman-plugin-discovery '
-            '--enable-foreman-proxy-plugin-discovery')
-        run('yum install -y {0}'.format(' '.join(packages)), warn_only=True)
         # Fetch the upstream-nightly FDI from upstream
         image_url = 'http://downloads.theforeman.org/discovery/nightly/fdi-image-latest.tar'  # noqa
-        run('wget {0} + -O - | tar x --overwrite -C /var/lib/tftpboot/boot'
+        run('wget -nv -O- {0} | tar x --overwrite -C /var/lib/tftpboot/boot'
```

Comment:
-nv 
"Turn off verbose without being completely quiet (use -q for that), which means that error messages and basic information still get printed."